sympy/functions/elementary/miscellaneous.py

```diff
@@ -380,7 +380,8 @@ def real_root(arg, n=None, evaluate=None):
 
 class MinMaxBase(Expr, LatticeOp):
     def __new__(cls, *args, **assumptions):
-        evaluate = assumptions.pop('evaluate', True)
+        from sympy.core.parameters import global_parameters
+        evaluate = assumptions.pop('evaluate', global_parameters.evaluate)
```

Comment:
Can you add a test for this in `sympy/functions/elementary/tests/test_miscellaneous.py`?

Reply:
Yes.

Reply:
I also added a new context `NoEvaluation` in the test file. This ensures that even if the new `test_minmax_no_evaluate` fails, it will not affect other tests by always setting the `gp.evaluate` back to `True` at its exit. Otherwise, we risk ending up in an environment where `gp.evaluate=False` for other tests if `test_minmax_no_evaluate` raises errors.

Reply:
Thx, I'll switch to that.